### DISCUSSION TITLE
一日土撥鼠 台北捷運_萬大中和線

### DISCUSSION THREAD

**Discussion by Lipunpun** [1101-GS4719-C/Python-Community-GS4719-C#1169] [General]
大家好我是土木系的學生，今天想和大家分享最近到萬大線的永和永平國小站,實際到現場看底下站體施工、潛盾隧道工程的參訪心得。

相信住在這附近的同學一定會覺得，馬路中間怎麼一直都在施工，到底是在做什麼？其實就是在做捷運，捷運工程包括地下站體的施工，以及潛盾隧道開挖，實際到現場才知道施工過程真的挺震撼的（為了尊重廠商，就不把照片傳上來了），地下四五層樓密密麻麻的支撐，還有工地特有的施工味道、噪音，還有潛盾機發進的過程。

整個參訪下來雖然要爬很多樓梯，但還是蠻有趣的，各種精密的儀器讓我以為我是在科技產業，但回過頭看到師傅黝黑的皮膚，還有紅紅的嘴巴，才突然驚醒原來我讀的是土木😆。

**Comment by Lipunpun** [comment]
潛盾隧道，是一種建造隧道的方法，這種方法適用在建造軟土地層或含水量很高的地層掘進隧道，大家可以參考這個。
https://zh.m.wikipedia.org/zh-tw/鑽挖式隧道

**Comment by Lipunpun** [comment]
地下連續牆，也翻譯為連續壁或槽壁，是在地下工程施工時建設的處於地表之下的鋼筋混凝土牆，用於支撐周圍的軟土層、擋水等目的。這項技術典型用於建築物的基坑的四壁、地下隧道或車站的防水、懸索橋錨碇錨坑施工、水庫大壩的防水心牆等等。大家可以參考這個網站。https://zh.m.wikipedia.org/zh-tw/地下连续墙

**Comment by tonyYZL** [comment]
從以前就很想親眼看看潛盾機工作的樣子，可惜我不是念土木相關的：（
只能在 youtube 的工業頻道看他們實地參訪的影片過過癮

**Comment by Lipunpun** [comment]
老師好～這堂課是土木系研究所的課，隧道工程學！可能不小心按到人工檢核～麻煩老師了！

**Reply by chia-kai-chang** [reply]
ok的
